src/main/daml/Daml/Finance/Interface/Claims/Dynamic/Instrument.daml

```diff
@@ -24,7 +25,7 @@ data View = View
 
 -- | Interface implemented by instruments that create Contingent Claims trees on-the-fly
 -- (ie., the tree is not stored on disk as part of a contract, but created and processed in-memory).
-interface Instrument requires BaseInstrument.I, Disclosure.I where
+interface Instrument requires BaseInstrument.I, Disclosure.I, SyntheticKey.I where
```

Comment:
Do we need to require `SyntheticKey` or is it sufficient that it's a transitive require (required from BaseInstrument.I)?

Reply:
we need to require `SyntheticKey`, it does not understand the transitive require.